docker/Dockerfile.base

```diff
@@ -3,7 +3,7 @@ FROM ubuntu:24.04
 ADD https://github.com/jgm/pandoc/releases/download/3.8.2.1/pandoc-3.8.2.1-1-amd64.deb /tmp/pandoc.deb
 
 RUN apt-get update -qq \
-    && apt-get install -y curl cmake wkhtmltopdf yaz libyaz-dev libmagickwand-dev \
+    && apt-get install -y curl cmake git wkhtmltopdf yaz libyaz-dev libmagickwand-dev \
```

Comment:
I had to add git to docker image used to run specs on Github, as we use github source in gemfile now